pkg/cmd/pkg/package_list.go

```diff
@@ -133,7 +134,7 @@ func newCmdPackageList(ctx api.Context) *cobra.Command {
 				return errors.New("there are currently no installed packages")
 			}
 
-			table := cli.NewTable(ctx.Out(), []string{"NAME", "VERSION", "APP", "COMMAND", "DESCRIPTION"})
+			table := cli.NewTable(ctx.Out(), []string{"NAME", "VERSION", "SELECTED", "APP", "COMMAND", "DESCRIPTION"})
```

Comment:
What about taking the opportunity and add a unit test for this cmd?

Reply:
This changes scope of this task